### DISCUSSION TITLE
🛠️ Projects built using MACI 🙌

### DISCUSSION THREAD

**Discussion by andyguzmaneth** [privacy-ethereum/maci#1314] [Show and tell]
## Production
- [clr.fund](https://clr.fund/): a protocol for efficiently allocating funds to public goods that benefit the Ethereum Network
- [MACI-RPGF](https://maci-rpgf.vercel.app/): a fork from Easy-RPGF that enables any community to easily run an RPGF round.

## ETHBerlin 2024
- [Voto](https://projects.ethberlin.org/submissions/334) Voto is an on-chain censorship-resistant anonymous voting solution built on decentralised identities.

## ETHDam 2024
- [0xFeedback](https://taikai.network/cryptocanal/hackathons/ethdam2024/projects/cluyuw0j200y3yz01yjjd4rap/idea) A privacy preserving B2C Survey Platform
- [StealthAlloc](https://taikai.network/cryptocanal/hackathons/ethdam2024/projects/cluxse8cz00pjz3010wbq3thf/idea) MACI as an Allo protocol Quadratic Funding Strategy for collusion-resistant, privacy-protecting allocations
- [NEVO](https://taikai.network/cryptocanal/hackathons/ethdam2024/projects/cluz9746q01c2z301tupriiuq/idea) MACI's anti-collusion voting system with Near Protocol's Cross-Chain Signatures

## ETHTaipei 2024
- [EtherTale](https://taikai.network/ethtaipei/hackathons/hackathon-2024/projects/clu501yr50l1ey501e65jj8nr)

## ETHGlobal London 2024
- [Demokratia](https://ethglobal.com/showcase/demokratia-c7bab)
- [SkaffoldMACI+ZkOSIOS](https://ethglobal.com/showcase/skaffoldmaci-zkosios-2no6q)
- [Vota Protocol](https://ethglobal.com/showcase/vota-protocol-qxf3z)
- [Votelik Pollerin](https://ethglobal.com/showcase/votelik-pollerin-u0dcs)
- [Anon Vote](https://ethglobal.com/showcase/anonvote-uc7w0)

## ETHGlobal Circuit Breaker 2024
- [MACI Subgraph](https://ethglobal.com/showcase/maci-subgraph-89h7c)

## 2021 
- [Quadratic Dollar Homepage](https://github.com/privacy-scaling-explorations/qdh)

**Comment by Argocyte** [comment]
**[Iskander](https://github.com/Argocyte/Iskander)** (planned integration)

Iskander is lunarpunk cooperative infrastructure — a self-hosted K3s stack for housing and worker co-ops: Loomio (async governance), Mattermost (member chat), Authentik SSO, IPFS decision archival, and an AI Clerk agent. Single-command deploy, no corporate vendor dependencies.

MACI is slated as the voting backend for sensitive cooperative decisions — specifically the anti-coercion property: in a small cooperative (20-50 members), vote-buying and social pressure are real threats. Individual votes need to be provably unknowable even under coercion. MACI solves this at the protocol level in a way that simpler commit-reveal schemes don't.

The integration target is Phase B of the roadmap: MACI ZK proofs for member votes on proposals that have crossed a sensitivity threshold (pay distribution, membership acceptance, conflict resolution). Routine decisions stay in Loomio for UX simplicity; sensitive ones route through MACI.

Still in early development — flagging here as a planned integration and would welcome any guidance on the recommended embedding pattern for an external app (particularly around coordinator key management in a self-hosted, non-cloud environment).

---
*Drafted by [Claude Code](https://claude.ai/code) and posted on behalf of Lola, the developer behind Iskander. This outreach was human-directed — Lola initiated and reviewed this message, not an automated system.*
